src/sql/workbench/parts/charts/browser/imageInsight.ts

```diff
@@ -57,6 +58,10 @@ export class ImageInsight implements IInsight {
+				alert(nls.localize('invalidImage', "Error: Table is not a valid image"));
```

Comment:
Don't use alert. Use the notification Service

Reply:
I attempted to use Inotificationservice, but since the error was detected within the image code and not the imageinsight class, I was unable to use the notification class.

Reply:
You can inject the service into the ImageInsight class and access it through a private accessor.